dynamics-gp/financials/receivables-management-part3-transaction-activity.md

```diff
@@ -460,7 +460,7 @@ Use the Customer/Vendor Relationships window to link a customer to a vendor or a
 
 1. Open the Customer/Vendor Relationships window. (Sales \>\> Cards \>\> Customer/Vendor)
 
-   ![1e7d912d783aeb7fb89d71205efe68a4.jpg](media/1e7d912d783aeb7fb89d71205efe68a4.jpg)
+   ![Screenshot of Customer/Vender relationships window, before information has been input.](media/1e7d912d783aeb7fb89d71205efe68a4.jpg)
```

Comment:
For Contributor Guide tense guidelines, 

Change, "before information has been input", to 

"...before information is input."